bindings/dotnet/CMakeLists.txt

```diff
@@ -127,18 +127,19 @@ add_custom_command(TARGET ${targetName}
 )
 
 # <PostBuildEvent>
-add_custom_command(TARGET ${targetName}
-                   # Run after all other rules within the target have been executed.
-                   POST_BUILD
-                   COMMENT "$<${configRelease}:[${targetName}] Building NuGet package (${platform}/$<CONFIG>)>"
-                   COMMAND "$<${configRelease}:${packCmd}>"
-                   WORKING_DIRECTORY "$<${configRelease}:${fullProjectDir}>"
-                   COMMAND_EXPAND_LISTS
-                   VERBATIM
-)
+#message(DEBUG "Adding POST_BUILD (dotnet pack)")
+#add_custom_command(TARGET ${targetName}
+#                   # Run after all other rules within the target have been executed.
+#                   POST_BUILD
+#                   COMMENT "$<${configRelease}:[${targetName}] Building NuGet package (${platform}/$<CONFIG>)>"
```

Comment:
Should these commented blocks be left here?

Reply:
For now, please, as it might be useful later. The research time for this might be useful later again. For sure I'll have to create some Confluence content for this eventually. Then we can remove it.